subsystems/actuators/breadboxArmController.py

```diff
@@ -73,45 +60,43 @@ def setFrontBottom(self):
         """
         sets the manipulator to the front bottom position
         """
-        self.setManipulator(self.setAnglesDegrees["frontBottom"], self.setArmLength["frontBottom"])
+        self.setManipulator(self.setAnglesDegrees["frontBottom"])
 
     def setFrontCenter(self):
         """
         sets the manipulator to the front center position
         """
-        self.setManipulator(self.setAnglesDegrees["frontMiddle"], self.setArmLength["frontMiddle"])
+        self.setManipulator(self.setAnglesDegrees["frontMiddle"])
```

Comment:
consider an enumerated type instead of frontMiddle

Reply:
name is from dictionary from yaml. Enum conversions would be decent am.ount more work to convert to I think